ansible/roles/pmix/tasks/main.yaml

```diff
@@ -0,0 +1,46 @@
+---
+
+- name: Include OS Family Dependent Vars
+  ansible.builtin.include_vars: '{{ item }}'
+  with_first_found:
+  - files:
+    - '{{ ansible_distribution|lower }}-{{ ansible_distribution_version }}.yml'
+    - '{{ ansible_distribution|lower }}-{{ ansible_distribution_major_version }}.yml'
+    - '{{ ansible_distribution|lower }}.yml'
+    - '{{ ansible_os_family|lower }}-{{ ansible_distribution_version }}.yml'
+    - '{{ ansible_os_family|lower }}-{{ ansible_distribution_major_version }}.yml'
+    - '{{ ansible_os_family|lower }}.yml'
+
+- name: Wait for DPKG Locks
+  ansible.builtin.shell: >
+    while fuser /var/lib/dpkg/{{ item }} >/dev/null 2>&1; do
+      sleep 5
+    done
+  with_items:
+  - lock
+  - lock-frontend
+  when:
+  - ansible_os_family == 'Debian'
+
+- name: Install PMIx dependencies
+  ansible.builtin.package:
+    name: '{{ pmix_dependencies }}'
+    state: present
+
+- name: Download PMIX
+  ansible.builtin.get_url:
+    url: https://github.com/openpmix/openpmix/releases/download/v{{ pmix_version }}/pmix-{{
+      pmix_version }}.tar.gz
+    dest: /tmp/pmix-{{ pmix_version }}.tar.gz
+    checksum: sha1:{{ sha1_v429 }}
+
+- name: Build PMIx
+  ansible.builtin.shell: |
+    cd /tmp
+    tar xzf pmix-{{ pmix_version }}.tar.gz
+    cd pmix-{{ pmix_version }}
+    ./configure --prefix=/usr >> conf.txt
```

Comment:
I'm sorry I didn't catch this earlier, but we should really be targeting `/usr/local` or similar. I'd recommend something like `pmix_path: {{ paths.install }}`

Reply:
We can't use `paths.install` as it is variable, and slurm only looks for pmix in `/usr` or `/usr/local`, I've updated to use the latter.